Core/Utilities/InstallTools.py

```diff
@@ -238,6 +239,7 @@ def loadDiracCfg( verbose = False ):
 mysqlRootUser = ''
 mysqlSmallMem = ''
 mysqlLargeMem = ''
+componentTypes = [ 'service', 'agent', 'executor' ]
```

Comment:
This is actually a global constant. For constants the convention is to use UPPER CASE letters, i.e.
componentTypes -> COMPONENT_TYPES